app/concepts/lockstep/query.rb

```diff
@@ -92,6 +92,35 @@ def include_object(*objects)
     end
   end
 
+  def in_batches(batch_size: 100)
```

Comment:
@shettytejas  Can we use `for_each` instead of `in_batches` to keep it consistent with rails framework?

Reply:
@vivekagb I've used this method from rails as the starting point.
https://api.rubyonrails.org/v6.1.7/classes/ActiveRecord/Batches.html#method-i-in_batches
would we still need it to be for_each?

Reply:
Ok double checked. This is what rails supports too. So we can go ahead with this.